installer/DetailsHandler.php

```diff
@@ -289,7 +288,7 @@ public function setConfigurationFromParameter($hostName, $hostPortOrSocket, $dat
         $sameUser = $this->IsSameOhrmUser($sameOhrmUser);
 
         //Admin and password
-        $_SESSION['defUser']['AdminUserName'] = $this->isFillInConfig($adminUserName, "Admin user name ");
+        $_SESSION['defUser']['AdminUserName'] = $this->isFillInConfigRegister($adminUserName, "Admin user name ");
```

Comment:
method name has grammatical issues.
Change to "isFilledInConfigRegister()"

Reply:
changed